contracts/Comptroller.sol

```diff
@@ -1021,6 +1036,44 @@ contract Comptroller is ComptrollerV3Storage, ComptrollerInterface, ComptrollerE
         allMarkets.push(CToken(cToken));
     }
 
+
+    /**
```

Comment:
Question on why to loop here-- couldn't the proposals simply call `_setMarketBorrowLimit` multiple times and save ourselves the difficulty of loops?

Reply:
When being called by governance, for sure would be simpler. The thought was that if the borrow limit guardian is updating limits on a regular basis through the gnosis multisig, it would be much more convenient to be able to do all of them in one tx. I wanted to have both a singular and multiple, but the contract size is too big then.